README.md

```diff
@@ -9,50 +9,73 @@ Solving problems with...
 - Dictionaries
 - Nested loops
 - Nested data structures
+- Pair-programming techniques
 
 ## Goal
 
 You and your friends enjoy watching things together online. Of course, everyone has seen different things, has different favorites, and different things they want to watch.
 
 You've been using a spreadsheet to compare everyone's watched list, favorites list, and watchlist, but it's been getting too cumbersome. In order to find things you've watched and your friends haven't watched, or things that your friends have watched and you haven't watched, you have to comb through the spreadsheet. You know that there are different ways we can get that information: we can use Python!
 
-For this project, you will be given some data structure that represents the things you've watched, favorited, and want to watch. The directions below will lead you to create a series of functions. These functions will modify the data, and implement features like adding and removing things between different lists. Other features include creating recommendations!
+For this project, you and your partner will be given some data structure that represents the things you've watched, favorited, and want to watch. The directions below will lead you and your partner to create a series of functions. These functions will modify the data, and implement features like adding and removing things between different lists. Other features include creating recommendations!
+
+## Pair Programming
+
+Utilize good pair programming practices. Refer to this [medium article](https://medium.com/@weblab_tech/pair-programming-guide-a76ca43ff389), and the lesson in Learn titled Intro to Pair Programming from Approaching a Problem if you need a refresher for some best practices. Switch _driver_ and _navigator_ roles often. When there is uncertainty or confusion, step away from the keyboard and discuss, plan, and document on paper or whiteboard before continuing.
 
 ## One-Time Project Setup
 
-Follow these directions once, a the beginning of your project:
+Follow these directions once, at the beginning of your project:
+
+*Only one member of the team should complete the following two steps steps:*
+
+1. In Github, click on the "Fork" button in github and fork the repository to your Github account.  This will make a copy of the project in your github account. 
```

Comment:
```suggestion
1. In Github, click on the "Fork" button in Github and fork the repository to your Github account.  This will make a copy of the project in your Github account. 
```

Reply:
```suggestion
1. In GitHub, click on the "Fork" button and fork the repository to your GitHub account.  This will make a copy of the project in your GitHub account. 
```